package.json

```diff
@@ -34,22 +34,23 @@
     "async-iterators": "^0.2.2",
     "eslint": "^3.12.2",
     "eslint-config-loopback": "^7.0.1",
-    "mocha": "^2.1.0",
-    "should": "^8.0.2"
+    "mocha": "^3.2.0",
+    "should": "^11.1.2"
   },
   "dependencies": {
-    "async": "~1.0.0",
+    "async": "~2.1.4",
```

Comment:
Do we know what were the breaking changes between async 1.x and 2.x? Can they cause a breaking change in juggler?